src/SFA.DAS.ApprenticeCommitments.Web.AcceptanceTests/Features/ConfirmYourTrainingProvider.feature

```diff
@@ -16,6 +16,7 @@ Scenario: The apprentice is authenticated and confirms the training provider
 	And the apprentice has not verified their training provider
 	And the apprentice confirms their training provider
 	When submitting the ConfirmYourTrainingProvider page
+	Then the apprenticeship is updated to show the confirmation
```

Comment:
Should we not have an equivalent check for when the User chooses to say No.

Reply:
Maybe, I'm not sure.  The only difference would be the value of the confirm state - which is being passed in from the test anyway.